c2corg_ui/static/js/editing/documentediting.js

```diff
@@ -655,13 +660,49 @@ app.DocumentEditingController.prototype.openModal = function(selector, sizem) {
   });
 };
 
-app.module.controller('appDocumentEditingController', app.DocumentEditingController);
+/**
+ * @param {appx.Document} doc Document attributes.
+ * @return {number}
+ */
+app.DocumentEditingController.prototype.presetQuality = function(doc) {
+  // Do nothing special in the standard editing controller.
+  // Will be overridden in inheriting controllers.
```

Comment:
Wouldn't it be better to return a default 1 value (i.e. draft)? The switch case below would be more consistent

Reply:
done